test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1137,8 +1137,8 @@ class test_ucp_wireup_unified : public test_ucp_wireup {
 public:
     static void get_test_variants(std::vector<ucp_test_variant>& variants)
     {
-        add_variant_with_value(variants, UCP_FEATURE_TAG,
-                               TEST_TAG | UNIFIED_MODE, "uni");
+        add_variant_with_value(variants, UCP_FEATURE_RMA,
```

Comment:
@yosefe  the test failed w/o this change, because:
Only tag feature requested -> RMA is disabled -> PUT explicitly disabled in DC -> DC is not preferred over RC.
Do you think there is a better way to fix this?

Reply:
probably need to update ucp_worker_iface_find_better to check only flags that are needed for UCP features, and not all flags, in here:
```c
        /* Check that another iface: */
        if (/* 1. Supports all capabilities of the target iface (at least),
             *    except ...CONNECT_TO... and KEEPALIVE-related caps. */
            ucs_test_all_flags(if_iter->attr.cap.flags, test_flags) &&
```